index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="//use.fontawesome.com/releases/v5.0.7/css/all.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <nav class="container">
+      <header class="toolbar">
+        <div class="logo">
+          <div class="placeholder">
+            <h3>Eshetu Melaku</h3>
+          </div>
+        </div>
+
+        <div class="normal-button">
+          <div id="icon-menu">
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+          </div>
+        </div>
+      </header>
+    </nav>
```

Comment:
- Kindly adjust the bottom spacing of the nav:

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/154272575-a4e6bc45-948b-482c-ac4d-aaf302770738.png) | ![image](https://user-images.githubusercontent.com/24703171/154272647-c536afd7-2b89-4ff5-88cd-082ba84959ac.png) |
decrease a height of the `nav` a little bit.

Reply:
thanks again, I just did that also.

Reply:
Addressed 🚀